aea/agent.py

```diff
@@ -110,7 +173,7 @@ def outbox(self) -> OutBox:
     @property
     def name(self) -> str:
```

Comment:
I keep this for backwards compatibility of the usage `self.name` and because there is no cost to keep it.